AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Campaigns/CampaignSummaryQueryHandlerTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿using AllReady.Areas.Admin.Features.Campaigns;
+using AllReady.Models;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace AllReady.UnitTest.Areas.Admin.Features.Campaigns
+{
+    public class CampaignSummaryQueryHandlerShould : InMemoryContextTest
+    {
+        protected override void LoadTestData()
+        {
+            var campaign = new Campaign
+            {
+                Id = 111,
+                ManagingOrganization = new Organization()
+            };
+
+            Context.Campaigns.Add(campaign);
+
+            Context.SaveChanges();
+        }
+
+        [Fact]
+        public async Task ReturnNull_WhenCampaignIsNotFound()
+        {
+            var handler = new CampaignSummaryQueryHandler(this.Context);
+
+            var result = await handler.Handle(new CampaignSummaryQuery { CampaignId = 222 });
+
+            result.ShouldBeNull();
+        }
+
+        [Fact]
+        public async Task ReturnCampaignSummaryViewModel_WhenCampaignExists()
+        {
+            var handler = new CampaignSummaryQueryHandler(this.Context);
+
+            var result = await handler.Handle(new CampaignSummaryQuery { CampaignId = 111 });
+
+            result.ShouldNotBeNull();
```

Comment:
this assertions does not assert what the test name says it should. The test name is `ReturnCampaignSummaryViewModel_WhenCampaignExists`, yet you're asserting that the result ShouldNotBeNull. You can change this test to assert it returns that specific view model by adding:
`Assert.IsType<CampaignSummaryViewModel>(result);`